backend/lucene/src/main/java/org/hibernate/search/backend/lucene/impl/LuceneBackendFactory.java

```diff
@@ -165,6 +171,11 @@ private MultiTenancyStrategy getMultiTenancyStrategy(ConfigurationPropertySource
 		}
 	}
 
+	private LuceneIdReaderWriter getIdReaderWriter(ConfigurationPropertySource propertySource) {
+		// TODO: read a property
```

Comment:
Say, `hibernate.search.backend.schema.id.strategy`, with two possible values: `lucene8` and `lucene9`?

I'd document this in a new subsection under https://docs.jboss.org/hibernate/search/6.2/reference/en-US/html_single/#backend-lucene-schema . Make sure to mention this can only be set at the backend level...

Reply:
ok, thanks 😃 I'll start cleaning up then 😃